sequencer/Cargo.toml

```diff
@@ -54,7 +54,7 @@ derivative = "2.2"
 derive_more = { workspace = true }
 dotenvy = { workspace = true }
 dyn-clone = { workspace = true }
-espresso-types = { path = "../types", features = ["testing"] }
+espresso-types = { path = "../types" }
```

Comment:
Maybe we shouldn't do this.

Reply:
fixed in https://github.com/EspressoSystems/espresso-sequencer/pull/2283/commits/56dcb1e57f5875c4259831ad83d241865d4997e7 i think